src/analog/CommandBinaryBuffer.ts

```diff
@@ -392,9 +392,9 @@ export interface ILegacyHourCounterWithDiff {
 }
 
 export interface IDataSegment {
-    sequence: number,
-    fragmentIndex: number,
-    fragmentsNumber: number,
+    segmentationSessionId: number,
```

Comment:
```suggestion
    sessionId: number,
    index: number,
    portionSize: number
```
maybe? 

we can discuss, it's already piece of segments